doc/docs/lexerdevelopment.rst

```diff
@@ -376,20 +376,20 @@ There are a few more things you can do with states:
       class ExampleLexer(RegexLexer):
           tokens = {
               'comments': [
-                  (r'/\*.*?\*/', Comment),
+                  (r'/\*(.|\n)*?\*/', Comment),
```

Comment:
Maybe it's more foolproof to set `(?s)` for the pattern after the bug we had here lately :)